changes/issue-1679-sync-file-writes

```diff
@@ -0,0 +1 @@
+* Sync critical file writes to disk.
```

Comment:
I would leave this changes file, but I would specify this is mostly an orbit change. @noahtalerman @zwass I guess one thing we haven't discussed is since we moved orbit into the repo, whether we would keep separate changelogs or how should we specify whether it's something that affects different places. What do you think?

Reply:
I added an `Orbit: ` prefix to the line to indicate this.